packages/web/stores/assets/transfer-ui-config.ts

```diff
@@ -129,15 +129,9 @@ export class ObservableTransferUIConfig {
   /** ### DEPOSIT/WITHDRAW FROM TABLE ROW
    *  User wants to transfer asset with counterparty IBC chain and coinDenom known. */
   @action
-  async transferAsset(
-    direction: TransferDir,
-    chainId: string,
-    coinDenom: string
-  ) {
+  async transferAsset(direction: TransferDir, coinDenom: string) {
```

Comment:
Let's update the comment above to say `with the direction and coinDenom known`.

Could be beneficial to add an additional note that the coinDenom is expected to be unique across chains.

Reply:
does that look good?